meinberlin/apps/budgeting/assets/BudgetingProposalListItem.jsx

```diff
@@ -12,14 +12,14 @@ export const BudgetingProposalListItem = (props) => {
   const { proposal, permissions, tokenvoteApiUrl } = props
   const safeLocale = props.locale ? props.locale : undefined
   const date = proposal.modified
-    ? `${updatedOnStr} ${toLocaleDate(
+    ? { updatedOnStr } + toLocaleDate(
```

Comment:
the {} are not needed, also it is missing a whitespace between `updatedOnStr` and `toLocaleDate` I think. So proper way would be `updatedOnStr + ' ' + toLocaleDate(`